operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys_on_PIT_Node.md

```diff
@@ -280,21 +293,23 @@ The Ceph image is used by the utility storage nodes.
    pit# mv -v *squashfs old
    ```
 
-1. Move new SquashFS image, kernel, and initrd into place.
+1. Move new SquashFS image, kernel, and `initrd` into place.
 
    ```bash
    pit# mv -v squashfs-root/squashfs/* .
    ```
 
-1. Update file permissions on initrd.
+1. Update file permissions on `initrd`.
 
    ```bash
    pit# chmod -v 644 initrd.img.xz
    ```
 
-1. Rename the new squashfs, kernel, and initrd to include a new version string.
+1. Rename the new SquashFS, kernel, and `initrd` to include a new version string.
 
-   If the old name of the squashfs was storage-ceph-0.1.69.squashfs, then its version was '0.1.69', so the newly created version should be renamed to include a version of '0.1.69-1' with an additional dash and a build iteration number of 1. This will help to track what base version was used.
+   If the old name of the SquashFS was `storage-ceph-0.1.69.squashfs`, then its version was '0.1.69',
+   so the newly created version should be renamed to include a version of '0.1.69-1' with an
+   additional dash and a build iteration number of 1. This will help to track what base version was used.
 
    ```bash
    pit# ls -l old/*squashfs
```

Comment:
Suggest changing 'common cleanup' section to something like 'deploy changes' to highlight the need to rebuild NCNs in certain flows. Applies to both docs in the PR.